src/Application.vala

```diff
@@ -19,29 +19,13 @@
 * Authored by: Felipe Escoto <felescoto95@hotmail.com>
 */
 
-public class Wallpaperize.Application : Granite.Application {
-    public const string PROGRAM_ID = "com.github.philip-scott.wallpaperize";
+public class Wallpaperize.Application : Gtk.Application {
     public const string PROGRAM_NAME = "Wallpaperize";
 
     construct {
         flags |= ApplicationFlags.HANDLES_OPEN;
 
```

Comment:
You may want to remove this extra new line, because there are only two lines in this constructor now.

Reply:
I don't really understand I already made these changes

Reply:
I'm sorry for my lack of explanation.

After you addressed my changes suggestion, line 25 to 29 of `Application.vala` should look like this:

```vala
    construct {
        flags |= ApplicationFlags.HANDLES_OPEN;

        application_id = "com.github.philip-scott.wallpaperize";
    }
```

Although this works well, I think it would be even better to remove the blank line (line 27) like this:

```vala
    construct {
        flags |= ApplicationFlags.HANDLES_OPEN;
        application_id = "com.github.philip-scott.wallpaperize";
    }
```

Reply:
Yes sure I'll fix this. I didn't understand you were speaking about blank lines :smile:

Reply:
This should be better now

Reply:
Yes, it looks better. Thank you!